<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thêm Sản Phẩm</string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string>color: rgb(149, 106, 214);</string>
   </property>
   <property name="text">
    <string>SẢN PHẨM MỚI</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_anh">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(255, 170, 255); border: 2px solid gray;</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_chon_anh">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(108, 187, 174); color: white; border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Chọn ảnh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_them">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>500</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(255, 255, 215); color: rgb(149, 106, 214); border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>THÊM</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_maSP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>TÊN SẢN PHẨM</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_giaSP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>55</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>MÃ SẢN PHẨM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>GIÁ THÀNH</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_soluongP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>SỐ LƯỢNG</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_tenSP">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_danhmucSP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
          background-color: white;
          border: 3px solid #3498db;  /* Viền xanh dương */
          border-radius: 10px;
          padding: 5px;
      }
      QComboBox::drop-down {
          border: none;
          background: transparent;
      }
      QComboBox::down-arrow {
          image: url(down-arrow.png); /* Thay bằng icon của bạn */
          width: 14px;
          height: 14px;
      }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>DANH MỤC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
   </property>
   <property name="text">
    <string>NHÀ CUNG CẤP</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbo_danhmucNCC">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
          background-color: white;
          border: 3px solid #3498db;  /* Viền xanh dương */
          border-radius: 10px;
          padding: 5px;
      }
      QComboBox::drop-down {
          border: none;
          background: transparent;
      }
      QComboBox::down-arrow {
          image: url(down-arrow.png); /* Thay bằng icon của bạn */
          width: 14px;
          height: 14px;
      }</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
